docs/cpu.md

```diff
@@ -0,0 +1,189 @@
+# CPU
+
+Cloud Hypervisor has many options when it comes to the creation of virtual
+CPUs. This document aims to explain what Cloud-Hypervisor is capable of and
```

Comment:
You change hyphenation in the middle of the paragraph. The project is called "Cloud Hypervisor" compare and contrast `git grep -c Cloud Hypervisor` with `git grep -c Cloud-Hypervisor`. If you want to refer to the binary do so as `cloud-hypervisor`

Reply:
Definitely!